AuthenticatorShared/UI/Vault/VaultItem/AuthenticatorKeyCapture/ManualEntryView.swift

```diff
@@ -117,13 +104,61 @@ struct ManualEntryView: View {
             })
         }
     }
+
+    /// A primary style button to trigger an `.addPressed(:)` action.
+    ///
+    private func addPrimaryButton(sendToBitwarden: Bool) -> some View {
```

Comment:
🎨 Near as I can tell these are identical aside from the primary/tertiary button style. Is there a way we could make this one function that passes that in?

Reply:
I've added a commit (with a quick assist from @victor-livefront 🙌) that reuses a lot more of the code. 👍 I also cleaned up the comments because these didn't have much on them previously.